FuzzySetLibTests/DistancesTest.cs

```diff
@@ -47,7 +47,10 @@ public void HammingDistanceTest()
                 {0.40f, 0.27f, 0.35f, 0.35f }
             };
 
-            var result = Distances.HammingSetDistance(Norms.Lukasiewicz, Implications.Lukasiewicz, Q, R);
+            var objectBounds = Approximations.ObjectBoundApproximation(Norms.Lukasiewicz, Implications.Lukasiewicz, Q, R);
+            var propertyBounds = Approximations.PropertyBoundApproximation(Norms.Lukasiewicz, Implications.Lukasiewicz, Q, R);
```

Comment:
imo te dwie tablice powinny być zahardkodowane - chodzi o to, żeby tutaj nie wołać funkcji aproksymujących, ale testować tylko dystanse, podobnie w teście poniżej.